config/checker-framework-suppressions/checker-nullness-optional-interning-suppressions.xml

```diff
@@ -140,17 +140,6 @@
     <lineContent>private InternalLoader()</lineContent>
   </checkerFrameworkError>
 
-  <checkerFrameworkError unstable="false">
-    <fileName>src/main/java/com/puppycrawl/tools/checkstyle/DefaultConfiguration.java</fileName>
-    <specifier>return</specifier>
-    <message>incompatible types in return.</message>
-    <lineContent>return children.toArray(</lineContent>
```

Comment:
Annotation must be placed on the method that has this line as it's return statement. Please take some time to read through the checker framework docs to understand this tool: https://checkerframework.org/manual/

Reply:
> Please take some time to read through the checker framework docs to understand this tool: https://checkerframework.org/manual/

On it sir.